.github/workflows/test.yml

```diff
@@ -11,8 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # switch to `8` when https://github.com/actions/setup-node/issues/27 is fixed
-        node: [8.16.2, 10, 12, 14]
+        node: [10, 12, 14]
```

Comment:
One last thing @kristerkari: this is the latest 10.x version. The same goes for the package.json engines version. I guess it's OK since we have the ESLint rules in place, but it's your call.

Reply:
That is ok. We shouldn't need to care about minor versions.